### PR TITLE
Adding 404 page

### PR DESCRIPTION
**Pull request by GregBorrelly**
Fixes https://github.com/babel/website/issues/1526

So not sure if I'm missing a step. I created a class base component that renders the HTML as it was on the legacy page.   Do we need a class base component? or is it ok if its pure;y functional? 

I haven't hooked up the component into anything, need your feedback to what the next steps should be.
``

### CODE REVIEW

**Comment by hzoo** [comment]
Not sure what fun thing we should do for the 404 page - maybe add play https://medium.com/@angustweets/hallelujah-in-praise-of-babel-977020010fad haha

**Comment by GregBorrelly** [comment]
so I have the live server running but when I type a url that's not there for example: http://localhost:3000/not-here I get `Cannot GET /not-here/index.html`

How does docusaurus know to respond with my 404 page? 

I do a lot of CSS animations at work. I can have an animation that on click expands to display the "praise to babel" I think that would be pretty cool, no?

**Comment by hzoo** [comment]
Oh ok I think the 404 is specifically from netlify itself :stuck:. I would just rename it to be a regular filename or directly check http://localhost:3000/404

**Comment by GregBorrelly** [comment]
makes sense,  I'll rename it, style it, and then name it back to 404.js

what did you think about the animation idea for the 404 page? too much?

**Comment by GregBorrelly** [comment]
Just added the styling to match the previous page

File: website/pages/en/404.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+
+class ErrorPage extends React.Component {
+    render() {
+        return (
+
+            <div className="error-page">
+                <div className="container">
+                    <div className="row">
+                        <div className="col-md-12 error-message">
```
Comment (hzoo) [inline website/pages/en/404.js]: Some of these styles won't mean anything anymore like `col-md-12` because we don't have bootstrap anymore

Comment (GregBorrelly) [reply website/pages/en/404.js]: I'll remove them, are we using another framework?

Comment (hzoo) [reply website/pages/en/404.js]: It's just whatever docusaurus provides and our own styles - The black heading was what we had before but I don't think we need that

Comment (GregBorrelly) [reply website/pages/en/404.js]: roger that, I want to work in something eye-catching as you mentioned before. I removed the bootstrap classes

**Comment by hzoo** [comment]
Ok I merged `docusaurus` into the `master` branch so you'll need to switch the target branch now (you can do this in github ui at the top)

**Comment by GregBorrelly** [comment]
Ok, I just switched branches to master.  What other tasks need to be done? or should I continue to expand on this 404 page?

**Comment by GregBorrelly** [comment]
@hzoo  IBM has a nice 404 page message
"Oops — that's not right!
Sorry, unless you really were hoping to find our 404 message, the page you were looking
for has been moved or deleted"

https://www.ibm.com/;www-01.ibm.com/events/wwe/indexconf/indexconf18.nsf/Registration.xsp?open

**Comment by hzoo** [comment]
Yeah now I'm thinking we should just have something simple, the google thing might not make that much sense to me given we have a search bar anyways

**Comment by hzoo** [comment]
If we can just have the footer at the bottom, and the contents taking up the whole space of the screen, and then a message like "not found" kind of thing, then we can update that it later

**Comment by GregBorrelly** [comment]
Sure, I will set it up

**Comment by GregBorrelly** [comment]
@hzoo made the update, there seem to be no conflicts but the netlify build failed.

**Comment by hzoo** [comment]
All good, I updated the build steps so it would fail if you didn't rebase. I'l push it up

**Pull request merged**